app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadFragment.java

```diff
@@ -280,13 +280,14 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 zimManageActivity.mSectionsPagerAdapter.libraryFragment.libraryAdapter.getFilter().filter(((ZimManageActivity) getActivity()).searchView.getQuery());
               }
             })
-            .setNegativeButton(R.string.no, (dialog, i) -> {
-              if (hasArtificiallyPaused) {
+                         .setNegativeButton(R.string.no, (dialogInterface, i) -> {
+                         if (hasArtificiallyPaused) {
                 hasArtificiallyPaused = false;
                 setPlayState(pause, position, DownloadService.PLAY);
               }
-            })
-            .show();
+            });
+        AlertDialog alertDialog = alertDialogBuilder.create();
```

Comment:
Are we actually using this ```alertDialog``` variable except in the next line ? If not, then replace the lines 289 and 290 with 
``` alertDialogBuilder.create().show()```

Reply:
Yeah ..we can do that.I will change it